src/components/Compare/CompareMain.tsx

```diff
@@ -30,73 +46,38 @@ const CompareMain = (props: {
   isHomepage?: boolean;
   currentCounty?: any;
   stateId?: string;
-  autoScroll?: boolean;
```

Comment:
FYI- The way autoScroll worked before was a little broken.  It would basically re-run on every render of the parent component, which wasn't desirable.  So I reworked it to use an effect, and just made the component automatically looks for /compare in the URL and scrolls itself into view if it's there (so `autoScroll` went away since the parent component doesn't need to pass it anymore).